precompiles/authorization/IICS20Authorization.sol

```diff
@@ -0,0 +1,70 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity >=0.8.17;
+
+import "../common/Types.sol";
+
+/// @author Evmos Team
+/// @title Authorization Interface
+/// @dev The interface through which solidity contracts will interact with smart contract approvals.
+interface IICS20Authorization {
+    /// @dev Emitted when an ICS-20 transfer authorization is granted.
+    /// @param grantee The address of the grantee.
+    /// @param granter The address of the granter.
+    /// @param allocations The Allocations authorized with this grant.
```

Comment:
I agree on using capital letters for specific types but AllocationS is not a type. I reformulated the sentence to use Allocation.
```suggestion
    /// @param allocations An array of Allocation authorized with this grant.
```